roles/os_hardening/tasks/sysctl.yml

```diff
@@ -68,6 +68,26 @@
       with_dict: '{{ sysctl_rhel_config }}'
       when: ansible_facts.distribution == 'Amazon'
 
+    - name: Change various sysctl-settings on Debian, look at the sysctl-vars file for documentation
+      sysctl:
+        name: '{{ item.key }}'
+        value: '{{ item.value }}'
+        state: present
+        reload: true
+        ignoreerrors: true
+      with_dict: '{{ sysctl_debian_config }}'
```

Comment:
all these different dicts look awfully redundant. What do you think about using one variable name like `sysctl_dist_config` and just use the same name for every linux distribution with a ` with_dict: '{{ sysctl_dist_config | default({}) }}'` in one task, that applies them?

Reply:
Yeah, I also thought about it. My solution was not much better than my current one, but your is great, I'll do it.
I just need to watch that it stays backwards-compatible.

Reply:
hmm, you are right, I wasn't considering backward compatibility. Either we introduce the new behavior only for Debian and Arch and keep RHEL as it is, and sometime later when we do a braking change we streamline it or we merge all tasks and do a variable override like with the OS specific vars for RHEL.